base/applications/rapps/include/rosui.h

```diff
@@ -116,46 +120,37 @@ class CPointerArray
         return DPA_Sort(m_hDpa, s_OnCompareItems, (LPARAM)this);
     }
 
-    INT Search(T* item, INT iStart, UINT uFlags)
+    INT Search(T* item, INT iStart = 0, UINT uFlags = 0)
     {
-        return DPA_Search(m_hDpa, item, 0, s_OnCompareItems, (LPARAM)this, 0);
+        return DPA_Search(m_hDpa, item, iStart, s_OnCompareItems, (LPARAM)this, uFlags);
```

Comment:
Make use of the `iStart` and `uFlags` parameters, giving them sane default values.